content/compute-at-sdsc.md

```diff
@@ -0,0 +1,68 @@
+
+#### Trial allocation
+
+Anybody can request a trial allocation on Comet with a quick 1 page justification and be approved within a day for 1000 core hours to be used within 6 month. This is useful to try Comet out, run some test jobs. See the [trial allocation page on the XSEDE website](https://portal.xsede.org/allocations/startup#trial).
+
+#### Campus champions
+
+Most US universities have a reference person that facilitates access to XSEDE supercomputers, it is often somebody in the Information Technology office or in a Chancellor of Research or a professor. This person is given a large amount of supercomputing hours on all XSEDE resources and local professors, postdocs and graduate students can request to be added to this allocation and use many thousands of core hours, depending on availability.
+Campus champions are currently available in 241 (!!) US institutions, [see the list on the XSEDE website](https://www.xsede.org/web/site/community-engagement/campus-champions/current).
+
+#### HPC@UC
+
+If you are at any of the University of California campuses, you have an expedited way of getting resources at SDSC.
+You can submit a request for up to 1 million core hours (more often ~500K core hours) on Comet on the [HPC@UC page](http://www.sdsc.edu/collaborate/hpc_at_uc.html). It just requires a 3 page justification and is answered within 10 business days. You are not eligible if your research group has an active XSEDE allocation.
+
+#### Startup allocation
+
+Startup allocations are really quick to prepare, they just require a 1 page justification and CV of the Principal Investigator and grant up to 50K core hours on Comet, if your research is funded by NSF/NASA/NIH remember to specify that. See the [startup page on XSEDE for more details](https://portal.xsede.org/allocations/startup).
+
+They are reviewed continously so you should be approved within a few days. Generally you are supposed to utilize the amount of hours within 1 year, but if your science project is funded for a longer period, you can request a multi-year allocation.
+
+#### XRAC allocation
+
+XRAC allocations are full fledged proposals, you can request up to a few million hours on Comet, here you must provide a detailed justification of the resources requested, demonstrate that your software is able to efficiently scale up in parallel, i.e. if in production you want to run on 100 nodes, you should run it on 5/10/50/100 nodes and check that performance does not degrade too much with increased parallelism.
+You should have performed those tests in a startup allocation.
+The XRAC requests are reviewed quarterly, see the [Research allocations page](https://portal.xsede.org/allocations/research), there is also a recorded webinar.
+
+## Triton Shared Computing Cluster
+
```

Comment:
A brief overview of what TSCC would be helpful. Maybe leave out the prices and refer to the TSCC website (just so the info doesn't become stale if prices change).